src/portal/src/i18n/lang/ko-kr-lang.json

```diff
@@ -0,0 +1,1943 @@
+        "END_USER_LICENSE": "최종 사용자 라이센스 계약",
+        "OPEN_SOURCE_LICENSE": "오픈소스/서드 파티 라이선스"
+    },
+    "START_PAGE": {
+        "GETTING_START": "",
+        "GETTING_START_TITLE": "시작하기"
+    },
+    "TOP_REPO": "인기 레포지토리",
+    "STATISTICS": {
+        "PRO_ITEM": "프로젝트",
+        "REPO_ITEM": "레포지토리",
+        "INDEX_PRIVATE": "프라이빗",
+        "INDEX_PUB": "퍼블릭",
+        "INDEX_TOTAL": "합계",
+        "STORAGE": "스토리지",
+        "LIMIT": "제한",
+        "STORAGE_USED": "스토리지 사용량"
+    },
+    "SEARCH": {
+        "IN_PROGRESS": "검색 중...",
+        "BACK": "뒤로"
+    },
+    "VULNERABILITY": {
+        "STATE": {
+            "OTHER_STATUS": "스캔되지 않음",
+            "QUEUED": "큐에 등록됨",
+            "ERROR": "로그 보기",
+            "SCANNING": "스캔 중",
+            "STOPPED": "스캔 중지 됨"
+        },
+        "GRID": {
+            "PLACEHOLDER": "스캐닝 결과를 찾을 수 없습니다!",
+            "COLUMN_ID": "취약점",
+            "COLUMN_SEVERITY": "심각도",
+            "COLUMN_PACKAGE": "패키지",
+            "COLUMN_PACKAGES": "패키지들",
+            "COLUMN_VERSION": "현재 버전",
+            "COLUMN_FIXED": "수정된 버전",
+            "COLUMN_DESCRIPTION": "설명",
+            "FOOT_ITEMS": "아이템",
+            "FOOT_OF": "of",
+            "IN_ALLOW_LIST": "CVE 허용 목록에 나열됨",
+            "CVSS3": "CVSS3"
+        },
+        "CHART": {
+            "SCANNING_TIME": "스캔 완료 시간:",
+            "SCANNING_PERCENT": "스캔 완료 퍼센테이지:",
+            "SCANNING_PERCENT_EXPLAIN": "스캔 완료 비율은 성공적으로 스캔한 이미지 수 / 이미지 인덱스 내에서 참조된 총 이미지 수로 계산됩니다.",
+            "TOOLTIPS_TITLE": "{{totalPackages}} {{package}}의 {{totalVulnerability}}가 {{vulnerability}}을 알고있습니다.",
+            "TOOLTIPS_TITLE_SINGULAR": "{{package}}의 {{totalVulnerability}} 가 {{vulnerability}}를 알고있습니다.",
+            "TOOLTIPS_TITLE_ZERO": "인식 가능한 취약점이 감지되지 않았습니다."
+        },
+        "SEVERITY": {
+            "CRITICAL": "중대한",
+            "HIGH": "높은",
+            "MEDIUM": "중간",
+            "LOW": "낮음",
+            "NONE": "없음"
+        },
+        "SINGULAR": "취약점",
+        "OVERALL_SEVERITY": "취약점 심각도:",
+        "NO_VULNERABILITY": "취약점 없음",
+        "PLURAL": "취약점",
+        "PLACEHOLDER": "취약점 필터",
+        "PACKAGE": "패키지",
+        "PACKAGES": "패키지들",
+        "SCAN_NOW": "스캔",
+        "SCAN_BY": "{{scanner}로 스캔",
+        "REPORTED_BY": "{{scanner}}로 보고 됨",
+        "NO_SCANNER": "스캐너 없음",
+        "TRIGGER_STOP_SUCCESS": "트리거 중지 스캔이 성공적으로 수행되었습니다",
+        "STOP_NOW": "스캔 중지"
+    },
+    "PUSH_IMAGE": {
+        "TITLE": "푸쉬 명령어",
+        "DOCKER": "Docker",
+        "PODMAN": "Podman",
+        "HELM": "Helm",
+        "CNAB": "CNAB",
+        "TAG_COMMAND_CHART": "이 프로젝트에 대한 차트(Chart) 패키징:",
+        "PUSH_COMMAND_CHART": "이 프로젝트에 차트 푸쉬:",
+        "PUSH_COMMAND_CNAB": "CNAB를 이 프로젝트에 푸쉬:",
+        "TOOLTIP": "이 프로젝트에 아티팩트를 푸시하기 위한 명령 참조입니다.",
+        "TAG_COMMAND": "이 프로젝트의 이미지에 태그를 지정:",
+        "PUSH_COMMAND": "이 프로젝트에 이미지 푸쉬:",
```

Comment:
Above, it is written as Push, but the spelling is not indicated here.
I think it would be a good idea to set rules and unify the notation of technical words.